src/main/java/jp/classmethod/aws/gradle/cloudformation/AmazonCloudFormationMigrateStackTask.java

```diff
@@ -164,16 +170,17 @@ private void updateStack(AmazonCloudFormation cfn) throws IOException {
 		if (isCapabilityIam()) {
 			Capability selectedCapability =
 					(useCapabilityIam == null) ? Capability.CAPABILITY_IAM : useCapabilityIam;
-			getLogger().error("Using policy: " + selectedCapability);
+			getLogger().info("Using policy: " + selectedCapability);
 			req.setCapabilities(Arrays.asList(selectedCapability.toString()));
 		}
 		
 		// If stack policy is specified, then use it
-		if (Strings.isNullOrEmpty(cfnStackPolicyUrl) == false) {
+		if (!Strings.isNullOrEmpty(cfnStackPolicyUrl)) {
```

Comment:
Our project does not use `!` operator because this is easy to overlook.
Please leave it as `== false`.